### ISSUE TITLE
Update Container Setup for Docker alternative

### ISSUE THREAD

**Issue by cchesser** [cchesser/java-perf-workshop#54]
[Docker recently updated their subscription terms](https://www.docker.com/blog/updating-product-subscriptions/), which mostly affects large businesses. We could keep its reference (free for personal use0, but it would make sense to update it to some recently identified alternatives that may be applicable to others who will be exposed to this difference.

**Comment by AnEmortalKid** [comment]
https://github.com/abiosoft/colima May be a good alternative to docker desktop. 

As a hunch, I feel anything in the realm of minikube kind k3ds May be a bit too much if containers are something people are learning for the first time.

**Comment by cchesser** [comment]
Yes, there definitely seems to be some better alternatives now (as the Apple M1 support seems to be more ubiquitous). [Rancher Desktop](https://rancherdesktop.io/) is another; however, it can be viewed to be a larger setup as well. [nerdctl via lima](https://github.com/containerd/nerdctl) and [podman](https://podman.io/) are others.
